draft-dekater-scion-controlplane.md

```diff
@@ -1461,22 +1486,111 @@ When the segment-request handler of a *core AS* control service receives a path
+
+
+**Fake Peering Link Announcement** <br>
+As an instance of a wormhole attack, an adversary can advertise fake peering links, thus offering short routes to many destination ASes within and outside its own ISD. Downstream ASes will likely have a policy of preferring paths with many peering links and thus are more likely to disseminate PCBs from the adversary. Similarly, endpoints are more likely to choose short routes that make use of peering links. However, a peering link can only be used if the neighboring AS also announces it. If the adversary is colluding with an external AS, a wormhole becomes possible. In the data plane, whenever the adversary receives a packet containing a fake peering link, it can transparently exchange the fake peering hop fields with valid hop fields to the colluding AS. To avoid detection of the path alteration by the receiver, the colluding AS can replace the added hop fields with the fake peering link hop fields the sender inserted.
+
+To defend against this kind of wormhole attacks, it is necessary to be able to detect these attacks. The current implementation of SCION is not able to do this (yet).
+
+
+## Denial of Service Attacks {#dos-cp}
+
+The beaconing process in the SCION control plane relies on control-plane communication: When ASes are propagating PCBs to downstream neighbor ASes, when they are registering PCBs as path segments at the core control services, or when they are looking up path segments at their own or a neighboring ISD's core, they constantly exchange control-plane messages with other ASes. DoS attacks, where attackers overload different parts of the IT infrastructure, may make it difficult to exchange these messages.
+
+SCION offers protection against volumetric DoS attacks, which aim to exhaust network bandwidth on links; in this case, ASes can switch to alternative paths that do not contain the congested links. However, it may be more difficult to avoid transport protocol attacks, where the attacker tries to exhaust the resources on a target server, such as a control service server, by opening many connections to this server. Possible means to mitigate this kind of DoS attacks are basically the same as for the current Internet, e.g., geo-blocking or using cookies.
 
-To be precised.
 
 
 # IANA Considerations
 
-TODO IANA considerations.
+SCION is an inter-domain routing architecture, which bases its inter-domain routing on the <ISD, AS> tuple. For more details, see [](#numbering).
+
+The ISD-AS number is a SCION-specific number. It consists of 64-bits, where the top 16 bits represent the 16-bit global identifier for the ISD, and the bottom 48 bits indicate the AS. Currently, the ISD and AS numbers are allocated by Anapaya, the Swiss-based provider of SCION-based networking software and solutions. However, in the future, the SCION Association will take care of the allocation - we are currently in the process of transferring this task from Anapaya to the Association.
```

Comment:
```suggestion
This document has no IANA actions.

The ISD and SCION AS number are SCION-specific numbers. They are currently allocated by Anapaya Systems, a provider of SCION-based networking software and solutions. This task is currently being transitioned  from Anapaya to the SCION Association.
```
Besides that, I'd add a reference to: https://docs.anapaya.net/en/latest/resources/isd-as-assignments 
As we (Association) take over the task, we will update this to our page.

Reply:
Also, I would delete from section 1.5.1 this text: 
``` 
A suitable mechanism to globally coordinate the assignation of ISD
   numbers does not yet exist.  However, we hope that in the future an
   organization such as ICANN or a regional Internet registry (e.g.,
   RIPE NCC) will take on the responsibility of assigning ISD and AS
   numbers.
   ```
   This is because the mechanism exists and it is Anapaya.. To get to have internet registries, we need the IANA section, which needs an IETF publication... So I'd just omit this text .
   
   Also, I'd add the same reference to Anapaya's page also there.

Reply:
@nicorusti: Thank you for your feedback. I have changed this draft, and also the other two drafts, accordingly.
Best
Corine